lte/gateway/deploy/agw_install_docker.sh

```diff
@@ -67,7 +67,17 @@ EOF
   echo "Install Magma"
   apt-get update -y
   apt-get upgrade -y
-  apt-get install curl zip python3-pip docker.io net-tools sudo docker-compose -y
+  apt-get install curl zip python3-pip net-tools sudo ca-certificates gnupg lsb-release -y
+  
+  mkdir -p /etc/apt/keyrings
+  curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /etc/apt/keyrings/docker.gpg
```

Comment:
Could we use long options here for better readability?
I.e. `curl --fail --silent --show-error --location` and `gpg --dearmor --output`

Reply:
Moved to a separate issue #14830